### PR TITLE
Add cards for Y to HH, H to tt

### PR DESCRIPTION
**Pull request by OzAmram**
These are the cards for Y to HH, H to tt. This is the 6+6 prong topology for the CASE analysis. Y mass scanned over 2, 3, and 5 TeV. H mass fixed to 400 GeV. 

cc @wpmccormack @thaarres

### CODE REVIEW

**Comment by OzAmram** [comment]
Hi @sihyunjeon and/or @agrohsje  would you mind taking a look at these? Thanks

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/BulkGraviton_hh_tttt/generateCards.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        print sampleName
+        if os.path.isdir(sampleName):
+            shutil.rmtree(sampleName)
+        os.makedirs(sampleName)
+        # Copy cards
+        shutil.copyfile(Ref_dir + 'run_card.dat',sampleName+'/'+sampleName+'_run_card.dat')
+        shutil.copyfile(Ref_dir+ 'madspin_card.dat',sampleName+'/'+sampleName+'_madspin_card.dat')
+        shutil.copyfile(Ref_dir+ 'extramodels.dat',sampleName+'/'+sampleName+'_extramodels.dat')
+        shutil.copyfile(Ref_dir + 'proc_card.dat',sampleName+'/'+sampleName+'_proc_card.dat')
+        shutil.copyfile(Ref_dir + 'customizecards.dat',sampleName+'/'+sampleName+'_customizecards.dat')
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/BulkGraviton_hh_tttt/generateCards.py:27]: shouldn't there be a customize card in the template dir?

**Comment by OzAmram** [comment]
Ok @agrohsje I updated the script to be clearer (but there is no functional change)

**Comment by agrohsje** [comment]
Looks good to me. @sihyunjeon  please check and merge if ok. Thanks!

**Pull request merged**